components/com_users/tmpl/registration/default.php

```diff
@@ -34,10 +35,19 @@
 					<?php if (isset($fieldset->label)) : ?>
 						<legend><?php echo Text::_($fieldset->label); ?></legend>
 					<?php endif; ?>
-					<?php echo $this->form->renderFieldset($fieldset->name); ?>
+					<?php foreach ($fields as $field): ?>			
+						<?php if ($field->name=="jform[captcha]"): ?> 
```

Comment:
```suggestion
						<?php if ($field->name=="jform[captcha]"): ?>
```

remove space at end of line